tchannel-example/src/main/java/com/uber/tchannel/ping/PingRequestHandler.java

```diff
@@ -27,12 +27,19 @@
 import com.uber.tchannel.api.ResponseCode;
 import com.uber.tchannel.api.handlers.JSONRequestHandler;
 
-public class PingDefaultRequestHandler extends JSONRequestHandler<Ping, Pong> {
+public class PingRequestHandler extends JSONRequestHandler<Ping, Pong> {
 
+    @Override
     public Response<Pong> handleImpl(Request<Ping> request) {
-        return new Response.Builder<>(new Pong("pong!"), request.getEndpoint(), ResponseCode.OK)
-                .setHeaders(request.getHeaders())
-                .build();
+
+        return new Response.Builder<>(
```

Comment:
this style is more confusing than the previous ones, also can we have teh style changes in a separate diff